### ISSUE TITLE
git lfs track "filename" hangs

### ISSUE THREAD

**Issue by frvge** [github/git-lfs#1304]
I downloaded the latest version of git lfs yesterday and installed it as root. Unfortunately when I now try to do `git lfs track "filename"` in a folder, it hangs. It uses a bit of CPU and seems to be stuck in the uwait state.

If I copy the file to another folder, it works.

Versions:
```
git version 2.4.2

git-lfs/1.2.1 (GitHub; freebsd amd64; go 1.6.2; git 79bedeb)

FreeBSD jenkins1.company.com 10.1-RELEASE-p19 FreeBSD 10.1-RELEASE-p19 #0: Sat Aug 22 03:55:09 UTC 2015     root@amd64-builder.daemonology.net:/usr/obj/usr/src/sys/GENERIC  amd64
```

truss output:
```
mmap(0x0,32768,PROT_READ|PROT_WRITE,MAP_PRIVATE|MAP_ANON,-1,0x0) = 34367737856 (0x8007a1000)
issetugid(0x8009a1a20,0x7fffffffefcd,0x40,0x0,0xffff8008009a2a53,0x0) = 0 (0x0)
lstat("/etc",{ mode=drwxr-xr-x ,inode=23,size=118,blksize=8192 }) = 0 (0x0)
lstat("/etc/libmap.conf",{ mode=-rw-r--r-- ,inode=58,size=145,blksize=4096 }) = 0 (0x0)
open("/etc/libmap.conf",O_CLOEXEC,01760)	 = 3 (0x3)
fstat(3,{ mode=-rw-r--r-- ,inode=58,size=145,blksize=4096 }) = 0 (0x0)
mmap(0x0,145,PROT_READ,MAP_PRIVATE,3,0x0)	 = 34367770624 (0x8007a9000)
close(3)					 = 0 (0x0)
lstat("/usr",{ mode=drwxr-xr-x ,inode=20,size=15,blksize=4096 }) = 0 (0x0)
lstat("/usr/local",{ mode=drwxr-xr-x ,inode=612,size=29,blksize=4096 }) = 0 (0x0)
lstat("/usr/local/etc",{ mode=drwxr-xr-x ,inode=171825,size=69,blksize=4608 }) = 0 (0x0)
lstat("/usr/local/etc/libmap.d",{ mode=drwxr-xr-x ,inode=171827,size=2,blksize=4096 }) = 0 (0x0)
open("/usr/local/etc/libmap.d",O_NONBLOCK|O_DIRECTORY|O_CLOEXEC,0165) = 3 (0x3)
__sysctl(0x7fffffffd0f8,0x2,0x8009a264c,0x7fffffffd0f0,0x0,0x0) = 0 (0x0)
fstatfs(0x3,0x7fffffffd118,0x8009a264c,0x7fffffffd0f0,0x0,0x0) = 0 (0x0)
getdirentries(0x3,0x8007a7000,0x1000,0x8007a6028,0x0,0x0) = 24 (0x18)
getdirentries(0x3,0x8007a7000,0x1000,0x8007a6028,0x18,0x0) = 0 (0x0)
close(3)					 = 0 (0x0)
mmap(0x0,36864,PROT_READ|PROT_WRITE,MAP_PRIVATE|MAP_ANON,-1,0x0) = 34367774720 (0x8007aa000)
munmap(0x8007a9000,145)				 = 0 (0x0)
access("/usr/local/lib/libz.so.6",0)		 ERR#2 'No such file or directory'
open("/var/run/ld-elf.so.hints",O_CLOEXEC,036537770) = 3 (0x3)
read(3,"Ehnt\^A\0\0\0\M^@\0\0\0\M-0\0\0"...,128) = 128 (0x80)
lseek(3,0x80,SEEK_SET)				 = 128 (0x80)
read(3,"/lib:/usr/lib:/usr/lib/compat:/u"...,176) = 176 (0xb0)
close(3)					 = 0 (0x0)
access("/lib/libz.so.6",0)			 = 0 (0x0)
open("/lib/libz.so.6",O_CLOEXEC,036537770)	 = 3 (0x3)
fstat(3,{ mode=-r--r--r-- ,inode=12474,size=89584,blksize=89600 }) = 0 (0x0)
mmap(0x0,4096,PROT_READ,MAP_PRIVATE|MAP_PREFAULT_READ,3,0x0) = 34367770624 (0x8007a9000)
mmap(0x0,2187264,PROT_NONE,MAP_PRIVATE|MAP_ANON|MAP_NOCORE,-1,0x0) = 34369843200 (0x8009a3000)
mmap(0x8009a3000,86016,PROT_READ|PROT_EXEC,MAP_PRIVATE|MAP_FIXED|MAP_NOCORE|MAP_PREFAULT_READ,3,0x0) = 34369843200 (0x8009a3000)
mmap(0x800bb8000,4096,PROT_READ|PROT_WRITE,MAP_PRIVATE|MAP_FIXED|MAP_PREFAULT_READ,3,0x15000) = 34372026368 (0x800bb8000)
munmap(0x8007a9000,4096)			 = 0 (0x0)
close(3)					 = 0 (0x0)
access("/usr/local/lib/libintl.so.8",0)		 = 0 (0x0)
open("/usr/local/lib/libintl.so.8",O_CLOEXEC,036537770) = 3 (0x3)
fstat(3,{ mode=-rw-r--r-- ,inode=15811481,size=55166,blksize=55296 }) = 0 (0x0)
mmap(0x0,4096,PROT_READ,MAP_PRIVATE|MAP_PREFAULT_READ,3,0x0) = 34367770624 (0x8007a9000)
mmap(0x0,2142208,PROT_NONE,MAP_PRIVATE|MAP_ANON|MAP_NOCORE,-1,0x0) = 34372030464 (0x800bb9000)
mmap(0x800bb9000,40960,PROT_READ|PROT_EXEC,MAP_PRIVATE|MAP_FIXED|MAP_NOCORE|MAP_PREFAULT_READ,3,0x0) = 34372030464 (0x800bb9000)
mmap(0x800dc3000,4096,PROT_READ|PROT_WRITE,MAP_PRIVATE|MAP_FIXED|MAP_PREFAULT_READ,3,0xa000) = 34374168576 (0x800dc3000)
munmap(0x8007a9000,4096)			 = 0 (0x0)
close(3)					 = 0 (0x0)
access("/usr/local/lib/libcrypto.so.8",0)	 = 0 (0x0)
open("/usr/local/lib/libcrypto.so.8",O_CLOEXEC,036537770) = 3 (0x3)
fstat(3,{ mode=-rw-r--r-- ,inode=36736178,size=2711596,blksize=131072 }) = 0 (0x0)
mmap(0x0,4096,PROT_READ,MAP_PRIVATE|MAP_PREFAULT_READ,3,0x0) = 34367770624 (0x8007a9000)
mmap(0x0,4534272,PROT_NONE,MAP_PRIVATE|MAP_ANON|MAP_NOCORE,-1,0x0) = 34374172672 (0x800dc4000)
mmap(0x800dc4000,2273280,PROT_READ|PROT_EXEC,MAP_PRIVATE|MAP_FIXED|MAP_NOCORE|MAP_PREFAULT_READ,3,0x0) = 34374172672 (0x800dc4000)
mmap(0x8011ee000,155648,PROT_READ|PROT_WRITE,MAP_PRIVATE|MAP_FIXED|MAP_PREFAULT_READ,3,0x22a000) = 34378539008 (0x8011ee000)
mmap(0x801214000,12288,PROT_READ|PROT_WRITE,MAP_PRIVATE|MAP_FIXED|MAP_ANON,-1,0x0) = 34378694656 (0x801214000)
munmap(0x8007a9000,4096)			 = 0 (0x0)
close(3)					 = 0 (0x0)
access("/usr/local/lib/librt.so.1",0)		 ERR#2 'No such file or directory'
access("/lib/librt.so.1",0)			 ERR#2 'No such file or directory'
access("/usr/lib/librt.so.1",0)			 = 0 (0x0)
open("/usr/lib/librt.so.1",O_CLOEXEC,036537770)	 = 3 (0x3)
fstat(3,{ mode=-r--r--r-- ,inode=12661,size=23832,blksize=24064 }) = 0 (0x0)
mmap(0x0,4096,PROT_READ,MAP_PRIVATE|MAP_PREFAULT_READ,3,0x0) = 34367770624 (0x8007a9000)
mmap(0x0,2121728,PROT_NONE,MAP_PRIVATE|MAP_ANON|MAP_NOCORE,-1,0x0) = 34378706944 (0x801217000)
mmap(0x801217000,20480,PROT_READ|PROT_EXEC,MAP_PRIVATE|MAP_FIXED|MAP_NOCORE|MAP_PREFAULT_READ,3,0x0) = 34378706944 (0x801217000)
mmap(0x80141c000,4096,PROT_READ|PROT_WRITE,MAP_PRIVATE|MAP_FIXED|MAP_PREFAULT_READ,3,0x5000) = 34380824576 (0x80141c000)
munmap(0x8007a9000,4096)			 = 0 (0x0)
close(3)					 = 0 (0x0)
access("/usr/local/lib/libthr.so.3",0)		 ERR#2 'No such file or directory'
access("/lib/libthr.so.3",0)			 = 0 (0x0)
open("/lib/libthr.so.3",O_CLOEXEC,036537770)	 = 3 (0x3)
fstat(3,{ mode=-r--r--r-- ,inode=12463,size=105248,blksize=105472 }) = 0 (0x0)
mmap(0x0,4096,PROT_READ,MAP_PRIVATE|MAP_PREFAULT_READ,3,0x0) = 34367770624 (0x8007a9000)
mmap(0x0,2248704,PROT_NONE,MAP_PRIVATE|MAP_ANON|MAP_NOCORE,-1,0x0) = 34380828672 (0x80141d000)
mmap(0x80141d000,102400,PROT_READ|PROT_EXEC,MAP_PRIVATE|MAP_FIXED|MAP_NOCORE|MAP_PREFAULT_READ,3,0x0) = 34380828672 (0x80141d000)
mmap(0x801635000,8192,PROT_READ|PROT_WRITE,MAP_PRIVATE|MAP_FIXED|MAP_PREFAULT_READ,3,0x18000) = 34383024128 (0x801635000)
mmap(0x801637000,45056,PROT_READ|PROT_WRITE,MAP_PRIVATE|MAP_FIXED|MAP_ANON,-1,0x0) = 34383032320 (0x801637000)
munmap(0x8007a9000,4096)			 = 0 (0x0)
close(3)					 = 0 (0x0)
access("/usr/local/lib/libc.so.7",0)		 ERR#2 'No such file or directory'
access("/lib/libc.so.7",0)			 = 0 (0x0)
open("/lib/libc.so.7",O_CLOEXEC,036537770)	 = 3 (0x3)
fstat(3,{ mode=-r--r--r-- ,inode=44196088,size=1631520,blksize=131072 }) = 0 (0x0)
mmap(0x0,4096,PROT_READ,MAP_PRIVATE|MAP_PREFAULT_READ,3,0x0) = 34367770624 (0x8007a9000)
mmap(0x0,3842048,PROT_NONE,MAP_PRIVATE|MAP_ANON|MAP_NOCORE,-1,0x0) = 34383077376 (0x801642000)
mmap(0x801642000,1523712,PROT_READ|PROT_EXEC,MAP_PRIVATE|MAP_FIXED|MAP_NOCORE|MAP_PREFAULT_READ,3,0x0) = 34383077376 (0x801642000)
mmap(0x8019b6000,49152,PROT_READ|PROT_WRITE,MAP_PRIVATE|MAP_FIXED|MAP_PREFAULT_READ,3,0x174000) = 34386698240 (0x8019b6000)
mmap(0x8019c2000,172032,PROT_READ|PROT_WRITE,MAP_PRIVATE|MAP_FIXED|MAP_ANON,-1,0x0) = 34386747392 (0x8019c2000)
munmap(0x8007a9000,4096)			 = 0 (0x0)
close(3)					 = 0 (0x0)
munmap(0x8007af000,16384)			 = 0 (0x0)
mmap(0x0,102400,PROT_READ|PROT_WRITE,MAP_PRIVATE|MAP_ANON,-1,0x0) = 34367795200 (0x8007af000)
sysarch(0x81,0x7fffffffe078,0x4,0x0,0xfffffffffedf8290,0xffffffff0fffffff) = 0 (0x0)
__sysctl(0x7fffffffe058,0x2,0x7fffffffe040,0x7fffffffe038,0x0,0x0) = 0 (0x0)
getrlimit(RLIMIT_STACK,{ cur=536870912,max=536870912 }) = 0 (0x0)
mprotect(0x7fffdffff000,536870912,PROT_READ|PROT_WRITE|PROT_EXEC) = 0 (0x0)
sigprocmask(SIG_BLOCK,SIGHUP|SIGINT|SIGQUIT|SIGKILL|SIGPIPE|SIGALRM|SIGTERM|SIGURG|SIGSTOP|SIGTSTP|SIGCONT|SIGCHLD|SIGTTIN|SIGTTOU|SIGIO|SIGXCPU|SIGXFSZ|SIGVTALRM|SIGPROF|SIGWINCH|SIGINFO|SIGUSR1|SIGUSR2,0x0) = 0 (0x0)
sigprocmask(SIG_SETMASK,0x0,0x0)		 = 0 (0x0)
readlink("/etc/malloc.conf",0x7fffffffd7a0,1024) ERR#2 'No such file or directory'
issetugid(0x80178d57e,0x7fffffffd7a0,0xffffffffffffffff,0x2,0x3d,0x0) = 0 (0x0)
mmap(0x0,4194304,PROT_READ|PROT_WRITE,MAP_PRIVATE|MAP_ANON,-1,0x0) = 34386919424 (0x8019ec000)
munmap(0x8019ec000,4194304)			 = 0 (0x0)
mmap(0x0,8384512,PROT_READ|PROT_WRITE,MAP_PRIVATE|MAP_ANON,-1,0x0) = 34386919424 (0x8019ec000)
munmap(0x8019ec000,2179072)			 = 0 (0x0)
munmap(0x802000000,2011136)			 = 0 (0x0)
sigprocmask(SIG_BLOCK,SIGHUP|SIGINT|SIGQUIT|SIGKILL|SIGPIPE|SIGALRM|SIGTERM|SIGURG|SIGSTOP|SIGTSTP|SIGCONT|SIGCHLD|SIGTTIN|SIGTTOU|SIGIO|SIGXCPU|SIGXFSZ|SIGVTALRM|SIGPROF|SIGWINCH|SIGINFO|SIGUSR1|SIGUSR2,0x0) = 0 (0x0)
sigprocmask(SIG_SETMASK,0x0,0x0)		 = 0 (0x0)
getpid()					 = 71407 (0x116ef)
__sysctl(0x7fffffffdbb0,0x2,0x8016418a0,0x7fffffffdbb8,0x0,0x0) = 0 (0x0)
__sysctl(0x7fffffffdad0,0x2,0x7fffffffdb10,0x7fffffffdb08,0x801431f6d,0xd) = 0 (0x0)
__sysctl(0x7fffffffdb10,0x3,0x80164032c,0x7fffffffdbb8,0x0,0x0) = 0 (0x0)
mmap(0x0,4194304,PROT_READ|PROT_WRITE,MAP_PRIVATE|MAP_ANON,-1,0x0) = 34393292800 (0x802000000)
thr_self(0x802006400,0x1,0x0,0x8007ae330,0x40,0x802016010) = 0 (0x0)
mmap(0x7fffffbfe000,4096,PROT_NONE,MAP_ANON,-1,0x0) = 140737484152832 (0x7fffffbfe000)
rtprio_thread(0x0,0x18997,0x7fffffffdb88,0x400000,0xffffffff,0x0) = 0 (0x0)
sysarch(0x81,0x7fffffffdb98,0x80163ff00,0x0,0xffffffff,0x0) = 0 (0x0)
sigaction(32,{ 0x80142aae0 SA_SIGINFO ss_t },0x0) = 0 (0x0)
sigprocmask(SIG_UNBLOCK,0x0,0x0)		 = 0 (0x0)
_umtx_op(0x7fffffffdb48,0x3,0x1,0x0,0x0,0x0)	 = 0 (0x0)
mprotect(0x0,0,PROT_NONE)			 = 0 (0x0)
getpid()					 = 71407 (0x116ef)
sigprocmask(SIG_BLOCK,SIGHUP|SIGINT|SIGQUIT|SIGILL|SIGTRAP|SIGABRT|SIGEMT|SIGFPE|SIGKILL|SIGBUS|SIGSEGV|SIGSYS|SIGPIPE|SIGALRM|SIGTERM|SIGURG|SIGSTOP|SIGTSTP|SIGCONT|SIGCHLD|SIGTTIN|SIGTTOU|SIGIO|SIGXCPU|SIGXFSZ|SIGVTALRM|SIGPROF|SIGWINCH|SIGINFO|SIGUSR1|SIGUSR2,0x0) = 0 (0x0)
sigprocmask(SIG_SETMASK,0x0,0x0)		 = 0 (0x0)
open("/dev/null",O_RDWR,00)			 = 3 (0x3)
close(3)					 = 0 (0x0)
sigprocmask(SIG_UNBLOCK,SIGPIPE,0x0)		 = 0 (0x0)
sigprocmask(SIG_SETMASK,SIGHUP|SIGINT|SIGQUIT|SIGILL|SIGTRAP|SIGABRT|SIGEMT|SIGFPE|SIGKILL|SIGBUS|SIGSEGV|SIGSYS|SIGPIPE|SIGALRM|SIGTERM|SIGURG|SIGSTOP|SIGTSTP|SIGCONT|SIGCHLD|SIGTTIN|SIGTTOU|SIGIO|SIGXCPU|SIGXFSZ|SIGVTALRM|SIGPROF|SIGWINCH|SIGINFO|SIGUSR1|SIGUSR2,0x0) = 0 (0x0)
sigaction(SIGPIPE,{ SIG_DFL SA_RESTART ss_t },{ SIG_DFL 0x0 ss_t }) = 0 (0x0)
sigprocmask(SIG_SETMASK,0x0,0x0)		 = 0 (0x0)
stat(".git",{ mode=drwxr-xr-x ,inode=24170918,size=14,blksize=131072 }) = 0 (0x0)
access("/usr/local/etc/gitconfig",4)		 ERR#2 'No such file or directory'
access("/root/.config/git/config",4)		 ERR#2 'No such file or directory'
access("/root/.gitconfig",4)			 = 0 (0x0)
open("/root/.gitconfig",O_RDONLY,0666)		 = 3 (0x3)
fstat(3,{ mode=-rw-r--r-- ,inode=25252510,size=92,blksize=4096 }) = 0 (0x0)
read(3,"[filter "lfs"]\n\tclean = git-lf"...,4096) = 92 (0x5c)
read(3,0x802060000,4096)			 = 0 (0x0)
close(3)					 = 0 (0x0)
access(".git/config",4)				 = 0 (0x0)
open(".git/config",O_RDONLY,0666)		 = 3 (0x3)
fstat(3,{ mode=-rw-r--r-- ,inode=24170940,size=92,blksize=4096 }) = 0 (0x0)
read(3,"[core]\n\trepositoryformatversio"...,4096) = 92 (0x5c)
read(3,0x802050000,4096)			 = 0 (0x0)
close(3)					 = 0 (0x0)
pipe(0x7fffffffe910)				 = 0 (0x0)
sigprocmask(SIG_BLOCK,SIGHUP|SIGINT|SIGQUIT|SIGILL|SIGTRAP|SIGABRT|SIGEMT|SIGFPE|SIGKILL|SIGBUS|SIGSEGV|SIGSYS|SIGPIPE|SIGALRM|SIGTERM|SIGURG|SIGSTOP|SIGTSTP|SIGCONT|SIGCHLD|SIGTTIN|SIGTTOU|SIGIO|SIGXCPU|SIGXFSZ|SIGVTALRM|SIGPROF|SIGWINCH|SIGINFO|SIGUSR1|SIGUSR2,0x0) = 0 (0x0)
fork()						 = 71408 (0x116f0)
sigprocmask(SIG_SETMASK,0x0,0x0)		 = 0 (0x0)
sigprocmask(SIG_SETMASK,SIGHUP|SIGINT|SIGQUIT|SIGILL|SIGTRAP|SIGABRT|SIGEMT|SIGFPE|SIGKILL|SIGBUS|SIGSEGV|SIGSYS|SIGPIPE|SIGALRM|SIGTERM|SIGURG|SIGSTOP|SIGTSTP|SIGCONT|SIGCHLD|SIGTTIN|SIGTTOU|SIGIO|SIGXCPU|SIGXFSZ|SIGVTALRM|SIGPROF|SIGWINCH|SIGINFO|SIGUSR1|SIGUSR2,0x0) = 0 (0x0)
sigaction(SIGINT,{ 0x80142af30 SA_RESTART|SA_SIGINFO ss_t },{ SIG_DFL 0x0 ss_t }) = 0 (0x0)
sigprocmask(SIG_SETMASK,0x0,0x0)		 = 0 (0x0)
sigprocmask(SIG_SETMASK,SIGHUP|SIGINT|SIGQUIT|SIGILL|SIGTRAP|SIGABRT|SIGEMT|SIGFPE|SIGKILL|SIGBUS|SIGSEGV|SIGSYS|SIGPIPE|SIGALRM|SIGTERM|SIGURG|SIGSTOP|SIGTSTP|SIGCONT|SIGCHLD|SIGTTIN|SIGTTOU|SIGIO|SIGXCPU|SIGXFSZ|SIGVTALRM|SIGPROF|SIGWINCH|SIGINFO|SIGUSR1|SIGUSR2,0x0) = 0 (0x0)
sigaction(SIGHUP,{ 0x80142af30 SA_RESTART|SA_SIGINFO ss_t },{ SIG_DFL 0x0 ss_t }) = 0 (0x0)
sigprocmask(SIG_SETMASK,0x0,0x0)		 = 0 (0x0)
sigprocmask(SIG_SETMASK,SIGHUP|SIGINT|SIGQUIT|SIGILL|SIGTRAP|SIGABRT|SIGEMT|SIGFPE|SIGKILL|SIGBUS|SIGSEGV|SIGSYS|SIGPIPE|SIGALRM|SIGTERM|SIGURG|SIGSTOP|SIGTSTP|SIGCONT|SIGCHLD|SIGTTIN|SIGTTOU|SIGIO|SIGXCPU|SIGXFSZ|SIGVTALRM|SIGPROF|SIGWINCH|SIGINFO|SIGUSR1|SIGUSR2,0x0) = 0 (0x0)
sigaction(SIGTERM,{ 0x80142af30 SA_RESTART|SA_SIGINFO ss_t },{ SIG_DFL 0x0 ss_t }) = 0 (0x0)
sigprocmask(SIG_SETMASK,0x0,0x0)		 = 0 (0x0)
sigprocmask(SIG_SETMASK,SIGHUP|SIGINT|SIGQUIT|SIGILL|SIGTRAP|SIGABRT|SIGEMT|SIGFPE|SIGKILL|SIGBUS|SIGSEGV|SIGSYS|SIGPIPE|SIGALRM|SIGTERM|SIGURG|SIGSTOP|SIGTSTP|SIGCONT|SIGCHLD|SIGTTIN|SIGTTOU|SIGIO|SIGXCPU|SIGXFSZ|SIGVTALRM|SIGPROF|SIGWINCH|SIGINFO|SIGUSR1|SIGUSR2,0x0) = 0 (0x0)
sigaction(SIGQUIT,{ 0x80142af30 SA_RESTART|SA_SIGINFO ss_t },{ SIG_DFL 0x0 ss_t }) = 0 (0x0)
sigprocmask(SIG_SETMASK,0x0,0x0)		 = 0 (0x0)
sigprocmask(SIG_SETMASK,SIGHUP|SIGINT|SIGQUIT|SIGILL|SIGTRAP|SIGABRT|SIGEMT|SIGFPE|SIGKILL|SIGBUS|SIGSEGV|SIGSYS|SIGPIPE|SIGALRM|SIGTERM|SIGURG|SIGSTOP|SIGTSTP|SIGCONT|SIGCHLD|SIGTTIN|SIGTTOU|SIGIO|SIGXCPU|SIGXFSZ|SIGVTALRM|SIGPROF|SIGWINCH|SIGINFO|SIGUSR1|SIGUSR2,0x0) = 0 (0x0)
sigaction(SIGPIPE,{ 0x80142af30 SA_RESTART|SA_SIGINFO ss_t },{ SIG_DFL SA_RESTART ss_t }) = 0 (0x0)
sigprocmask(SIG_SETMASK,0x0,0x0)		 = 0 (0x0)
close(4)					 = 0 (0x0)
read(3,0x7fffffffe914,1)			 = 0 (0x0)
close(3)					 = 0 (0x0)
```

**Comment by technoweenie** [comment]
Hi @frvge, `git lfs track` is hanging because it's touching affected files. Basically, when you run`git lfs track "*.psd"`, it's going to touch every file that matches that pattern so it gets converted to an LFS pointer. This ensures that the repository files are converted to LFS at the time of tracking.

I definitely think the `track` command should provide some feedback about what it's doing, and include an option to skip this.

**Comment by frvge** [comment]
@technoweenie , if I remember correctly, it was only 1 filename. When copied to another folder, it was tracked almost instantly. The original was Jenkins' war file. I'm not sure if that war file is actively being read/written to and that that might affect git lfs's capability to "track/touch" it.

**Comment by ttaylorr** [comment]
Hi, @frvge! I am hoping to ship #1344 on Tuesday, which should hopefully provide more insight into what is going on and if LFS is getting stuck touching something.

#1344 also comes with the `--no-touch` flag, which will skip the touching all together. 😄

**Comment by ttaylorr** [comment]
Hi again! I just merged #1344, which should provide some more insight into what's going on when you execute the `git lfs track` command.

We decided against the `--no-touch` flag, but you can provide `--dry-run`, which will show you what the track command would do, and inspect the output to see which file `track` is hanging on.
